src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerDetailResourceTest.kt

```diff
@@ -359,8 +332,25 @@ class PrisonerDetailResourceTest : QueueIntegrationTest() {
   fun `no-terms query should match all prisoners in the specified location`() {
     detailSearch(
       detailRequest = PrisonerDetailRequest(prisonIds = listOf("MDI")),
-      expectedCount = 7,
-      expectedPrisoners = listOf("A7089EY", "A7089FA", "A7089FB", "A7090AA", "A7090AB", "A7090BB", "A1090AA"),
+      expectedPrisoners = listOf("A1090AA", "A7089EY", "A7089FA", "A7089FB", "A7090AA", "A7090AB", "A7090BB"),
     )
   }
+
+  private fun detailSearch(
+    detailRequest: PrisonerDetailRequest,
+    expectedPrisoners: List<String> = emptyList(),
+  ) {
+    val response = webTestClient.post().uri("/prisoner-detail")
+      .body(BodyInserters.fromValue(gson.toJson(detailRequest)))
+      .headers(setAuthorisation(roles = listOf("ROLE_GLOBAL_SEARCH")))
+      .header("Content-Type", "application/json")
+      .exchange()
+      .expectStatus().isOk
+      .expectBody(RestResponsePage::class.java)
+      .returnResult().responseBody
+
+    assertThat(response.content).extracting("prisonerNumber").containsExactlyElementsOf(expectedPrisoners)
+    assertThat(response.content).size().isEqualTo(expectedPrisoners.size)
```

Comment:
`expectedCount` was always the same as the expected number of prisoners list so unnecessary